css/style.css

```
@@ -15,5 +14,178 @@ body {
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
+.header {
+  display: flex;
+  justify-content: space-around;
+}
+#logo {
+  width: 100px;
+  height: 50px;
+}
+
+.navbar ul {
+  display: flex;
+  flex-direction: row;
+  list-style-type: none;
+}
+.navbar ul li {
+  padding: 10px;
+}
+
+.list-nav li a {
+  text-decoration: none;
+  color: black;
+  opacity: 0.6;
+}
+.list-nav li a:hover {
+  color: rgb(248, 92, 44);
+}
+.list-nav li a:active {
+  color: black;
+}
+#menu {
+  display: none;
+}
+.section-1 {
+  background-image: url("/img/first-background.jpg");
+  height: 730px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  color: white;
+  margin: 0px;
+}
+.section-1 h2 {
+  font-weight: 70;
+  font-size: 3.5rem;
+  margin-block-start: 0em;
+  margin-block-end: 0em;
+}
+.section-1 p {
+  font-weight: 80;
+  font-size: 2rem;
+}
+.section-1 button {
+  background-color: rgb(248, 92, 44);
+  height: 2.5em;
+  width: 8.5em;
+  border: none;
+  border-radius: 0.5em;
+  margin-top: 15px;
+}
+
+.section-2 {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin: 50px;
+}
+.section-2 h2 {
+  text-align: center;
+}
+
+.h-font {
+  font-family: "Roboto", sans-serif;
 
+  font-size: 40px;
+  text-align: center;
+}
+.section3 {
+  display: flex;
+  flex-direction: row;
 
+  justify-content: center;
+}
+.section3 div {
+  margin: 65px;
+}
+.section3 div img {
+  width: 130px;
+  height: 165px;
+}
+.footer {
+  display: flex;
+
+  flex-direction: column;
+  align-items: center;
```

```suggestion
  text-align: center;
```
Like my other comment, the children elements of `.footer` are already in a column so we don't need flex. We can use `text-align: center` to centre the elements instead